torch/nn/functional.py

```diff
@@ -2484,10 +2484,10 @@ def _pointwise_loss(lambd, lambd_optimized, input, target, reduction='mean'):
         return lambd_optimized(expanded_input, expanded_target, _Reduction.get_enum(reduction))
 
 
-def _smooth_l1_loss(input, target):
+def _huber_loss(input, target, delta=1.):
```

Comment:
nit: changing the name may break other codes, even though this is a private function

Reply:
Changed back to `_smooth_l1_loss`